api/kyverno/v1/common_types.go

```diff
@@ -422,6 +426,80 @@ type PodSecurityStandard struct {
 	Images []string `json:"images,omitempty" yaml:"images,omitempty"`
 }
 
+// CEL defines the CEL expression which is used to apply the validation.
+type CEL struct {
+	// Expressions contain CEL expressions which is used to apply the validation.
+	Expressions []CELExpression `json:"expressions,omitempty" yaml:"expressions,omitempty"`
+
+	// ParamKind specifies the kind of resources used to parameterize this policy.
+	// If absent, there are no parameters for this policy and the param CEL variable will not be provided to validation expressions.
+	// If ParamKind refers to a non-existent kind, this policy definition is mis-configured and the FailurePolicy is applied.
+	// +optional
+	ParamKind *ParamKind `json:"paramKind,omitempty" yaml:"paramKind,omitempty"`
+
+	// AuditAnnotations contains CEL expressions which are used to produce audit annotations for the audit event of the API request.
+	// +optional
+	AuditAnnotations []AuditAnnotation `json:"auditAnnotations,omitempty" yaml:"auditAnnotations,omitempty"`
+}
+
+// CELExpression specifies the CEL expression which is used to apply the validation.
+type CELExpression struct {
```

Comment:
Can we directly use the Kubernetes API `Validation` type, instead of defining our own?